recipes/binutils/all/conanfile.py

```diff
@@ -181,16 +179,19 @@ def package_info(self):
 
         # v1 exports
         bindir = os.path.join(self.package_folder, "bin")
-        self.env_info.PATH.append(bindir)
-        self.env_info.PATH.append(absolute_target_bindir)
+        self.runenv_info.prepend_path("PATH", bindir)
+        self.runenv_info.prepend_path("PATH", absolute_target_bindir)
```

Comment:
```suggestion
        self.env_info.prepend_path("PATH", bindir)
        self.env_info.prepend_path("PATH", absolute_target_bindir)
```

runenv_info and buildenv_info automatically appends bin folder to PATH, but on Conan 1.x it does not happen with env_info.

Reply:
Please, revert this change.

Reply:
Sorry, I don't follow. Migration guide says that `env_info` was replaced by `runenv_info` and `buildenv_info`. If conan v2 automatically adds `bin` to build and run environment I need some way to do it for conan v1.

Reply:
Correct, for Conan v2 is deprecated, but in CCI we need to support both versions. So in case you are running Conan 1.x, you no longer will be able to find binutils tools in case using `self.run(..., run_environment=True)`

Reply:
Got it, thank you. I added `env_info` back